src/locales/es-ES.json

```diff
@@ -19,6 +19,7 @@
   "strategiesPage": "Estrategias",
   "space": "Espacio",
   "spaces": "Espacios",
+  "verifiedSpace": "Verified space",
```

Comment:
Can you remove this from all translation files, we only need it in `default.json`